webapp/security/views.py

```diff
@@ -71,6 +71,13 @@ def notice(notice_id):
         "releases": notice.releases,
         "cves": notice.cves,
         "references": notice.references,
+        "related_notices": [
+            {
+                "id": n.id,
+                "package_list": ", ".join(n.package_list),
+            }
+            for n in notice.related_notices
```

Comment:
Use `notice` instead of `n` please.

Reply:
Actually `related_notice` has `notice` is already defined

Reply:
Done.